custom_components/econnect_metronet/devices.py

```diff
@@ -74,6 +73,15 @@ def sectors(self):
     def alerts_v2(self):
         yield from self._inventory.get("alerts", {}).items()
 
+    def reset(self):
```

Comment:
Write a better docstring and test the method. Ideally, it should be clear that you need to add IDs of your collections in this function when they get reset.

Reply:
This needs to be removed as it changes how the fix is addressed. It's better to separate this change from this refactoring.